bc-wikitemplate.md

```diff
@@ -1,87 +1,103 @@
 
-## Installer
+
+### Crash verification
+- [ ] Verify these [crash issues](https://github.com/brave/brave-browser/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A%22Releasable+builds+0.55.x%22+label%3Acrash+is%3Aclosed++label%3AQA%2FNeeded) have not regressed 
+- [ ] Verify these [webview crash issues](https://github.com/brave/brave-browser/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A%22Releasable+builds+0.55.x%22+label%3Acrash/webview+is%3Aclosed++label%3AQA%2FNeeded) have not regressed 
+
+
+### Installer
 
 - [ ] Check that installer is close to the size of last release.
```

Comment:
@bbondy will we be mostly using stub installers? If that's the case, we'll need to change this as it's not really a valid check as the application we'll be pulled/installed from a remote server. If we're going to support both regular binaries and stub installers, we should add another case similar to the following:

* Install brave using the stub installer and ensure the installed profile is similar in size to the previous version

Reply:
I retained that test for the offline installer for Windows. These setup related tests are blocked on https://github.com/brave/omaha/issues/4